icu4c/source/tools/genrb/parse.cpp

```diff
@@ -1191,9 +1194,11 @@ realParseTable(ParseState* state, TableResource *table, char *tag, uint32_t star
 
         if (token == TOK_CLOSE_BRACE)
         {
+#if WARN_EMPTY_TABLE
             if (!readToken) {
                 warning(startline, "Encountered empty table");
             }
+#endif
```

Comment:
```suggestion
            if (!readToken && isVerbose()) {
                warning(startline, "Encountered empty table");
            }
```
... is a better fix I think.

Reply:
\*sigh\* You're right.  And my question to get one of these simple changes through in one try continues...